src/map/Mapbox.ts

```diff
@@ -6,6 +6,10 @@ import { SetPoint, SetSelectedPath } from '@/actions/Actions'
 import { Popup } from '@/map/Popup'
 import { FeatureCollection, LineString } from 'geojson'
 import { Bbox, Path } from '@/api/graphhopper'
+import mapboxgl from "mapbox-gl";
+window.mapboxgl = mapboxgl;
```

Comment:
> Maybe it would be even better not to render it at all? For long routes the svg inside the control can become quite large.

Does the browser do rendering work even for when display is none?

Reply:
At least one can see the elements in the inspector. So, somehow they are part of the dom.

Reply:
Ok then maybe we should use `user agent` or similar to determine whether or not the widget should be shown and do not add the map control in the first place.